pods/main-chart-list/main-chart.vm.ts

```diff
@@ -11,3 +11,10 @@ export const createDefaultSampleChart = (): ChartInfoVm => ({
   picture: "static/images/default_chart.png",
   chartPageRoute: "espana-covid-marzo",
 });
+
+export const createSpainPopulationChart = (): ChartInfoVm => ({
+  title: "Spain population by provinces",
+  description: "Spain population by provinces (data updated on 1st January 2020)",
```

Comment:
We have to include the source where we extracted the data

Reply:
True. I'm gonna fix that quickly.

Reply:
I'm thinking we should create a new field on the interface ***ChartInfoVm*** (**pods/main-chart-list/main-chart.vm.ts**) called *source*, puttin the source outside *description* field. Then we could use that new filed on component **chart-info-card.tsx** as a link and not like plain text (or also plain text but separeted from description).  What do you think about it?

I adjoint the current state of **chart-info-card.tsx**:
![image](https://user-images.githubusercontent.com/33926302/85553301-1a078600-b624-11ea-9aed-7706dec4888c.png)

Reply:
Source and Tags as well (tag array) we need it to get good SEO support

Reply:
Something like this (?):
![image](https://user-images.githubusercontent.com/33926302/85568941-6bb70d00-b632-11ea-93d6-72c4ea068ecf.png)

Reply:
That's great, I would add this as an issue and take it in a separate branch